docs/source/specs/openapi.json

```diff
@@ -1972,9 +1972,15 @@
             "type": "array",
             "items": {
               "type": "object",
-              "example": {
-                "detail": "Not Found.",
-                "status": 404
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "example": "Not found."
+                },
+                "status": {
+                  "type": "string",
+                  "example": "403"
```

Comment:
Should this still be 404? It's just the spec example, so it probably doesn't matter, but wanted to make sure.